janus_core/calculations/geom_opt.py

```diff
@@ -56,6 +57,8 @@ class GeomOpt(BaseCalculation):
         Set force convergence criteria for optimizer in units eV/Å. Default is 0.1.
     steps : int
         Set maximum number of optimization steps to run. Default is 1000.
+    symmetrize : bool
```

Comment:
Just to double check US/UK ise/ize consistency in API.

Reply:
I somewhat regret the choice but we generally use ize e.g. optimize/minimize, so this should be consistent.

(From a very quick look I think the only thing I've been inconsistent with is "Initialise" in docstrings, but variables are all ok I think)